src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_multiDiffusion_panorama.py

```diff
@@ -0,0 +1,708 @@
+# Copyright 2023 Omar Bar Tal and The HuggingFace Team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2023 Omer Bar Tal and The HuggingFace Team. All rights reserved.
```

Reply:
Also, the name of the script needs to be changed accordingly: either `pipeline_stable_diffusion_panorama.py` or `pipeline_stable_diffusion_panorama.py`. 

What ever we decide [here](https://github.com/huggingface/diffusers/pull/2393/files#r1109415926).